distance_factor: 3.0
angle_factor: 0.1
min_distance: 0.6
max_distance: 2.5
